components/uid.rst

```diff
@@ -310,13 +313,14 @@ entity primary keys::
     namespace App\Entity;
 
     use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Bridge\Doctrine\Types\UlidType;
     use Symfony\Component\Uid\Ulid;
 
     class Product
     {
         /**
          * @ORM\Id
-         * @ORM\Column(type="ulid", unique=true)
+         * @ORM\Column(type=UlidType::NAME, unique=true)
```

Comment:
@marcelsiegert there is also the `setParameter` example where `uuid` is provided as type. I think that could also be replaced with the constant instead.